kubernetes/config-maps/concent-api-worker/local_settings.py.j2

```diff
@@ -18,7 +18,7 @@ DEBUG_INFO_IN_ERROR_RESPONSES = {{ allow_debug_info_in_error_responses }}
 
 LOGGING['handlers']['console']['level'] = DEBUG
 
-{% if signing_service_on_the_cluster %}
+{% if allow_signing_service_authentication %}
 USE_SIGNING_SERVICE = True
 {% endif %}
```

Comment:
When `USE_SIGNING_SERVICE` is `False`, the configuration must contain the private key. Otherwise Concent won't be able to sign transactions.

Reply:
Done.